### ISSUE TITLE
Update to windrose-card 1.7.0 broke my windrose card

### ISSUE THREAD

**Issue by kllerbees** [aukedejong/lovelace-windrose-card#76]
All was good until update.  I have tried every thing to get back the wind flags with no luck.
Before update it looked like the example without the wind gust 
![windrose](https://github.com/user-attachments/assets/27c0d3e2-0466-4991-8844-72a0cff5bdaf)

type: custom:windrose-card
title: Wind Rose
data_period:
  hours_to_show: 4
max_width: 400
refresh_interval: 5
windspeed_bar_location: right
windspeed_bar_full: false
wind_direction_entity:
  entity: sensor.redtail_trl_wind_direction
  direction_unit: letters
  use_statistics: false
  show_current_direction_arrow: true
  direction_compensation: 0
windspeed_entities:
  - entity: sensor.redtail_trl_wind
    name: WIndspeed
    speed_unit: mph
    speed_range_step: 32
    use_statistics: true
    wind_direction_count: 16
    render_relative_scale: false
output_speed_unit: mph
input_speed_unit: mph
speed_range_beaufort: false
rose_lines: rgb
windrose_draw_north_offset: 0
cardinal_direction_letters: NSEW
matching_strategy: speed-first
center_calm_percentage: true
speed_range_max: 50
speed_range_step: 5

**Comment by aukedejong** [comment]
Can you try version 1.8.1?
If that doesn't work, can you post errors in the browser console?

**Comment by kllerbees** [comment]
I tried version 1.8.1 I get no errors just no NaN% where wind direction flags and scales should be.
Using a Entities Card configuration see jpg everything is reporting to HA fine.

----- Original Message -----

**Comment by aukedejong** [comment]
Can you try 1.8.2?
Handling of wind direction states is improved. Hopefully fixes the problems you experience.

**Comment by kllerbees** [comment]
I loaded older versions and still don't see direction flags. I loaded your newer version of 1.8.2 and nothing. I am thinking that the upgrade to 1.7.0 wasn't the problem. I get my weather entities from Weatherlink and there have been some recent changes there. I now have two wind directions entities one in 8 points of the compass in letters and one in degrees. I have played with both and can't get it to come up right. It was working at one time three weeks ago. Thangs for the help 

----- Original Message -----
